tests/_support/Publishers/TestPublisher.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Tests\Support\Publishers;
+
+use CodeIgniter\Publisher\Publisher;
+use RuntimeException;
+
+final class TestPublisher extends Publisher
+{
+	/**
+	 * Fakes an error on the given file.
+	 *
+	 * @return $this
```

Comment:
```suggestion
```

Reply:
Oops! this used to be an instance method 😅